src/sage/rings/number_field/number_field.py

```diff
@@ -11273,33 +11311,35 @@ def _log_gen(self, x):
             sage: K._log_gen(CDF(a^4))
             4
 
-            sage: zeta105 = CC(exp(2*pi*i/105))                                         # optional - sage.symbolic
-            sage: K.<a> = CyclotomicField(105, embedding=zeta105^13)                    # optional - sage.symbolic
-            sage: zeta105^13, CC(a)                                                     # optional - sage.symbolic
+            sage: # needs sage.symbolic
+            sage: zeta105 = CC(exp(2*pi*i/105))
+            sage: K.<a> = CyclotomicField(105, embedding=zeta105^13)
+            sage: zeta105^13, CC(a)
             (0.712376096951345 + 0.701797902883992*I,
              0.712376096951345 + 0.701797902883991*I)
-            sage: K._log_gen(zeta105^26)                                                # optional - sage.symbolic
+            sage: K._log_gen(zeta105^26)
             2
-            sage: K._log_gen(zeta105)                                                   # optional - sage.symbolic
+            sage: K._log_gen(zeta105)
             97
-            sage: zeta105, CC(a^97)                                                     # optional - sage.symbolic
+            sage: zeta105, CC(a^97)
             (0.998210129767735 + 0.0598041539450342*I,
              0.998210129767736 + 0.0598041539450313*I)
-            sage: K._log_gen(zeta105^3)                                                 # optional - sage.symbolic
+            sage: K._log_gen(zeta105^3)
             81
-            sage: zeta105^3, CC(a)^81                                                   # optional - sage.symbolic
+            sage: zeta105^3, CC(a)^81
             (0.983929588598630 + 0.178556894798637*I,
              0.983929588598631 + 0.178556894798635*I)
 
             sage: K.<a> = CyclotomicField(5, embedding=None)
```

Comment:
shouldn't we add `# needs sage.symbolic` here ? it's not the same bloc due to blank line

Reply:
Above needed `sage.symbolic` because of `exp`, `pi`, `i`. This one here doesn't